benchmarks/src/models/architectures/ssd.py

```diff
@@ -15,11 +15,14 @@ def initialize_ssd(
     pretrained_backbone: bool = True,
     trainable_backbone_layers: Optional[int] = 5,
     **kwargs: Any,
-) -> nn.Module:
-    return ssd300_vgg16(
-        pretrained=pretrained,
-        trainable_backbone_layers=trainable_backbone_layers,
-        num_classes=num_classes,
-        pretrained_backbone=pretrained_backbone,
-        **kwargs,
+) -> Tuple[nn.Module, float]:
+    return (
+        ssd300_vgg16(
```

Comment:
Can this not be saved as and returned as "model" like the others?

Reply:
All the other models require something to happen to the `model` object; for SSD, there's no modifications necessary, so there is no need to create the extra variable.

Do you reckon it increases readability to do it with the `model` object? If so, I am happy to create it.

Reply:
If it's not needed then I think the readability is fine without